front/app/containers/Admin/projects/project/projectHeader/PublicationStatus/index.tsx

```diff
@@ -104,7 +105,26 @@ const PublicationStatus = ({ project }: { project: IProject }) => {
 
   // Only display the component if the project has been published at least once
   if (!project.data.attributes.first_published_at) {
-    return null;
+    return (
+      <Box
+        display="flex"
+        alignItems="center"
+        gap="8px"
+        as="span"
+        padding="4px 8px"
+        border={`1px solid ${colors.borderDark}`}
+        borderRadius={stylingConsts.borderRadius}
+      >
+        <Icon
+          name={publicationStatusIcon}
+          fill={publicationStatusIconColor}
+          width="20px"
+        />
+        <Text color="coolGrey600" m="0px">
+          {formatMessage(statusMessage)}
+        </Text>
+      </Box>
```

Comment:
If not too difficult, it would be nice to limit the code duplication here.

Reply:
It's not difficult but it differs ever so slightly so in this case, I would rather have duplication that an abstraction to avoid having to untangle unrelated UI elements moving forward :)